po/wesnoth/nl.po

```diff
@@ -7188,15 +7188,15 @@ msgstr "Schakel een kant naar/van AI controle."
 #, fuzzy
 #| msgid "do not translate the on/off^[<side> [on/off]]"
 msgid "do not translate the on/off/full^[<side> [on/off/full]]"
-msgstr "vertaal de on/off^[<zijde>[on/off]] niet"
+msgstr "[<zijde>[on/off]]"
```

Comment:
This is missing the 'full', which seems a common problem for translations of this string (although most don't have the prefix).

What can we do to work better with the translators? It's clear that a lot of fuzzy strings have been simply marked as up-to-date.

Reply:
Another case of a fuzzy translation that needs to be updated. Leaving as-is.